src/app/shared/formly/types/date-picker-field/date-picker-field.component.scss

```diff
@@ -0,0 +1,8 @@
+@import 'variables';
+
+.input-group {
+  ::ng-deep .ngb-dp-weekday {
```

Comment:
`::ng-deep` is deprecated, isn't it? Global styling should be used here.

Reply:
Looking at our CoP Angular discussion results we agreed on using `::ng-deep` in this way in component styles:
```
 :host ::ng-deep
```
That way it gets some kind of encapsulation back.

But @dhhyi is right this could also be a global style, especially since we probably want all our date pickers to look the same not just the once used via formly. What about adding `.ngb-dp-weekday` to `src\styles\global\forms.scss`?